docs/guide/installation-procedure.md

```diff
@@ -304,6 +304,17 @@ open ~/Applications/Webots.app    # to launch Webots using the open command
 
 Alternatively, you can double-click on the Webots icon to launch it.
 
+#### Trouble Shooting for Mac M1 Users
+
+If you are geting errors like these
+`...(mach-o file, but is an incompatible architecture (have (arm64), need (x86_64)))`
+This is caused by Rosetta loading Webots under the x86 architecture instead of the native ARM. Unless if you plan on using x86 binaries with your Webots simulator, make sure to turn off Rosetta. Sometimes it may try to open the app using Rosetta by default, which may cause issues with running your code inside the simulator. 
```

Comment:
```suggestion
This is likely caused by Rosetta loading Webots under the x86 architecture instead of the native ARM. Unless you really want to use x86 binaries with your Webots simulation, make sure to turn off Rosetta. Sometimes macOS may try to open the app using Rosetta by default, which may cause issues when it comes to run robot controllers with dependencies on ARM libraries.
```